homework9/src/components/ImageBackground/ImageBackground.jsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import { clsx } from "clsx";
+import "./imageBackground.styles.css";
+
+const ImageBackground = ({ image, className, alt }) => {
+  return (
+    <div className={clsx(className, "img-bg")}>
+      <div></div>
```

Comment:
do u need that empty `div`?

Reply:
Yes. That is the what border in the image